### DISCUSSION TITLE
Is there a way to change which icons and banners will be used as default for games that don't have one?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#466] [Q&A]
I don't like those blank icons.

**Comment by neuromancer** [comment]
Use the "Image URL" and  "Vertical Image URL" in the game properties.

**Reply by ghost** [reply]
Does that only work on a per game base?
Cause I wanna change how all icons and banners look for all games that normally would have blank ones and that they will get applied automatically.

**Comment by tkashkin** [comment]
There's no option to change default icons, but you can use `Download game images` option to automatically set images for games that don't have them.

**Reply by ghost** [reply]
That download game image option is cool,but doing it manually takes to long,so is there a way to set that it searched for the game cover and applies the first image to all games automatically?
